cluster/addons/fluentd-gcp/fluentd-gcp-image/google-fluentd.conf

```diff
@@ -16,14 +16,14 @@
   type tail
   format none
   time_key time
-  path /var/lib/docker/containers/*/*-json.log
-  pos_file /var/lib/docker/containers/gcp-containers.log.pos
+  path /varlog/containers/*/*.log
```

Comment:
Nope. It is not missing. The directory that is mapping in from the outer host is called `/varlog/`.
Perhaps I should call it something else :-)